reader.go

```diff
@@ -283,9 +284,9 @@ func getFieldFromSlice(out []uint64, slice []uint64, width, offset uint64) ([]ui
 		out = append(out, field)
 	}
 
-	if widthWords > 1 {
-		out = ShiftSliceOfUint64Left(out, int(offset%64))
-	}
+	// if widthWords > 1 {
```

Comment:
get64BitsFieldFromSlice actually takes care of the shifting logic already, so this is not necessary